enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -485,5 +490,7 @@ private void resendOtp(final OnboardingProcessEntity process, final String otpCo
             logger.warn("OTP code resend failed, error: {}", e.getMessage(), e);
             throw new OnboardingOtpDeliveryException(e);
         }
+
+        auditService.auditOnboardingProvider(process, "Re-sent activation OTP for user: {}", userId);
```

Comment:
I would prefer `Resent` over `Re-sent`, the first spelling is used frequently.

Reply:
No problem, I will change it. I was not sure what to choose.